interview-questions.md

```diff
@@ -6,42 +6,66 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-Type your initials here to confirm you read the instructions:
+Type your initials here to confirm you read the instructions: js
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 Your answer:
 
 Researched answer:
 
-1. What is git? What is the difference between git and Github?
+<!-- 1. What is git? What is the difference between git and Github? -->
 
-Your answer:
+<!-- Your answer: 
+like a database for your code to maintain and see cahnges as it grows, as the code is being used by multiple coders --> 
 
-Researched answer:
+<!-- git is the main distributin warehouse of the code while github you can create repositories and can be beneficial with teams and project management -->
 
-2. What is the difference between an array's index and its value?
 
-Your answer:
+<!-- Researched answer: 
 
-Researched answer:
+version control system through repositories and making updates within your code through time and how its being used.   -->
+<!-- Git is the version control system itself, while GitHub is a platform that hosts Git repositories and provides additional collaboration and project management features on top of Git.  -->
 
-3. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
+<!-- ///////////////////////////////////////// -->
+
+ <!-- 2. What is the difference between an array's index and its value?   -->
+
+<!-- Your answer: 
+an array is a special type of object used for storing multiple values in a single variable. 
+
+index is the placeholder within the array starting in th zero postion 
+var array = [index0, index1, index2, index3]the array is waht holds the data that is going to be tested upon.  -->
+ 
+<!-- Researched answer: -->
+<!-- the index of an array identifies the position of an element within the array, while the value represents the actual data stored at that position. The combination of index and value allows programmers to access and manipulate the elements stored in an array efficiently. -->
 
+ 
+
+<!-- ///////////////////////////////////////////////////// -->
+
+3. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
+<!-- 
 Your answer:
 
-Researched answer:
+ arrays store a range of data to use in to accees to information within the index. string are the number of characters within the array  -->
+
+Researched answer:Arrays are used to group together data of the same type. A string is a sequence of characters, or it can be defined as a collection of objects of the char data type
+
+<!-- ////////////////////////////////////////////////////////// -->
```

Comment:
Good job!  I think studying vocab will help you overall.  It can help you understand concepts, speak about your code more coherently, and really help to improve your code! All from vocab. 😀. Try elaborating a bit more on these researched answers so you exhibit your understanding.  You can do this, Judas! 🎉